app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
```

Comment:
I like JSDocs (and docs in other languages), but I have a rule which is to only include documentation that's useful. Docs can really help clarify the contract for methods, but they also take up valuable page space that could be taken up with easy-to-read code. So I omit docs when the function is easy to understand, or self-explanatory (or likewise for the parameter and return value.

Also, if you are going to do JSDocs, I think they follow JavaDoc format, so they are enclosed in `/** */` - at least, that's what we've done elsewhere.